<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>edit_tree</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(229, 227, 221);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: rgb(202, 195, 212);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>60</width>
        <height>661</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	color: rgb(229,227,221);
	line-height:10px;
	font: 22pt &quot;Cascadia Mono&quot;;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGraphicsView" name="treePic"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="old">
         <property name="styleSheet">
          <string notr="true">    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    background: transparent; 
    border: 1px solid gray; 
	border-color: #E5E5E5;
	background-color: #FFFFFF;
</string>
         </property>
         <property name="placeholderText">
          <string>Old species names</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="new1">
         <property name="styleSheet">
          <string notr="true">    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    background: transparent; 
    border: 1px solid gray; 
	border-color: #E5E5E5;
	background-color: #FFFFFF;
</string>
         </property>
         <property name="placeholderText">
          <string>New species names</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="modify">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: rgb(188, 149, 117);
border:none;
border-radius:7px;
color:white}
QPushButton::hover {
    background-color: rgb(226, 179, 141);
}

QPushButton::pressed {
    background-color: rgb(158, 125, 99);
}</string>
         </property>
         <property name="text">
          <string>Modify</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="rootTxt">
           <property name="styleSheet">
            <string notr="true">    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    background: transparent; 
    border: 1px solid gray; 
	border-color: #E5E5E5;
	background-color: #FFFFFF;
</string>
           </property>
           <property name="placeholderText">
            <string>Please enter the species you want to set as an ancestor</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="flash">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: rgb(188, 149, 117);
border:none;
border-radius:7px;
color:white}
QPushButton::hover {
    background-color: rgb(226, 179, 141);
}

QPushButton::pressed {
    background-color: rgb(158, 125, 99);
}</string>
         </property>
         <property name="text">
          <string>Flash</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
